src/hooks/Guilds/web3/useJsonRpcProvider.ts

```diff
@@ -1,17 +1,6 @@
-import { useWeb3React } from '@web3-react/core';
-import { providers } from 'ethers';
-import { useContext, useMemo } from 'react';
-import { MultichainContext } from '../../../contexts/MultichainProvider';
+import { useProvider } from 'wagmi';
 
-export default function useJsonRpcProvider(chainId?: number) {
-  const { providers: multichainProviders } = useContext(MultichainContext);
-
-  const { library, chainId: walletChainId } = useWeb3React();
-  return useMemo(
-    () =>
-      (!chainId || walletChainId === chainId) && library
-        ? new providers.Web3Provider(library.currentProvider)
-        : multichainProviders[chainId],
-    [library, chainId, walletChainId, multichainProviders]
-  );
+export default function useJsonRpcProvider() {

```

Comment:
Shouldn't we remove the entire `useJsonRpcProvider` and use `useProvider` across the app?